pkg/universal_sigver/client_test.go

```diff
@@ -0,0 +1,181 @@
+package universal_sigver
+
+import (
+	"context"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/crypto"
+	signer_pkg "github.com/layer-3/clearsync/pkg/signer"
+	"github.com/layer-3/clearsync/pkg/smart_wallet"
+	"github.com/layer-3/clearsync/pkg/userop"
+	"github.com/layer-3/clearsync/pkg/userop/testing/local_blockchain"
+	"github.com/shopspring/decimal"
+	"github.com/stretchr/testify/require"
+)
+
+// FIXME: rename `local_blockchain` to something better
+func TestVerify(t *testing.T) {
+	ctx := context.Background()
+
+	// 1. Setup node, contracts, signer
+	node := local_blockchain.NewEthNode(ctx, t)
+	contracts := local_blockchain.SetupContracts(ctx, t, node)
+	eoa, err := local_blockchain.NewAccountWithBalance(ctx, big.NewInt(42), node)
```

Comment:
@dpatsora , I was referring to these functions. `local_blockchain` package name does not seem to provide any context of what these functions do.
I think, we should rename it to something like `aa-test-infra`.
@maxpushka, @dpatsora  , what do you think?

Reply:
Also, you can see that other functions (namely, `NewBundler` and `SendNative`) are being used in this test.

Reply:
aa-test-infra +1

Reply:
I vote for `infra/local` and `infra/public` to better align with Go code style recommendations.
(my longer comment on the topic is [here](https://github.com/layer-3/clearsync/pull/224#discussion_r1528335762))